modules/post/windows/gather/forensics/memorydump.rb

```diff
@@ -0,0 +1,230 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+#
+# This Module has an embred Belkasoft RAM Capturer version of 2018-08
+# Available at https://belkasoft.com/ram-capturer
+##
+
+#
+# Forensic memory dump using Belkasoft RAM Capturer
+#
+# Helvio Junior (M4v3r1cK) m4v3r1ck.hjr [at] gmail.com
+#    https://www.helviojunior.com.br
+#
+# Other modules repository: https://github.com/helviojunior/metasploit_modules
+#
+# Standard Library
+#
+require 'tmpdir'
+
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+  include Msf::Post::Windows::UserProfiles
+
+  def initialize(info={})
+    super( update_info( info,
+      'Name'          => 'Windows Gather Memory Dump Imaging',
+      'Description'   => %q{This module will perform a memory dump of remote machine},
+      'License'       => MSF_LICENSE,
+      'Platform'      => ['win'],
+      'SessionTypes'  => ['meterpreter'],
+      'Author'        => ['Helvio Junior (M4v3r1cK) <m4v3r1ck.hjr[at]gmail.com>']
+    ))
+    register_options(
+      [
+        OptString.new('RAMCAPTURE_PATH',[true,'Path of Belkasoft RAM Capturer',''])
```

Comment:
As this is the path on the local system, `OptString` can be changed to `OptPath`.

Reply:
Is the folder of Belkasoft, to try to make this more clear i made a post check explanation, if file was not found or was in a wrong structure the module will shows this messages:

[-] File not found: /tmp/x64/msvcp110.dll
[!] This module depeds on Belkasoft RAM Capturer application available at https://belkasoft.com/ram-capturer
[*] You need to download end extract the Belkasoft RAM Capturer at this system and set RAMCAPTURE_PATH option to ram capture path
[*] For example: If the ram capture was extracted at /root/, this will have a structure of directories similar at showed bellow
[*] /root/
[*] ├── Ram_Capturer
[*] │   ├── x64
[*] │   │   ├── RamCapture64.exe
[*] │   └── x86
[*] │       └── RamCapture.exe
[!] At this case you need to set RAMCAPTURE_PATH to /root/Ram_Capturer
[*] Post module execution completed